peripherals/rcc/rcc_common.yaml

```diff
@@ -0,0 +1,121 @@
+# Common bits of RCC peripheral
+
+RCC:
+  CR:
+    PLLRDY:
+      _read:
+        Unlocked: [0, "PLL unlocked"]
+        Locked: [1, "PLL locked"]
+    PLLON:
+      "Off": [0, "PLL off"]
+      "On":  [1, "PLL on" ]
+    CSSON:
+      "Off": [0, "Clock security system disabled (clock detector OFF)"]
+      "On": [1, "Clock security system enable (clock detector ON if the HSE is ready, OFF if not)"]
+    HSEBYP:
+      NotBypassed: [0, "HSE crystal oscillator not bypassed"]
+      Bypassed: [1, "HSE crystal oscillator bypassed with external clock"]
+    HSERDY:
+      _read:
+        NotReady: [0, "HSE oscillator not ready"]
+        Ready: [1, "HSE oscillator ready"]
+    HSEON:
+      "Off": [0, "HSE oscillator OFF"]
+      "On": [1, "HSE oscillator ON"]
+    HSICAL: [0, 255]
+    HSITRIM: [0, 31]
+    HSIRDY:
+      _read:
+        NotReady: [0, "HSI oscillator not ready"]
+        Ready: [1, "HSI oscillator ready"]
+    HSION:
+      "Off": [0, "HSI oscillator Off"]
+      "On": [1, "HSI oscillator On"]
+  CFGR:
+    PPRE*:
+      DIV1: [0, "HCLK not divided"] # Same for [0, 8]
```

Comment:
I personally don't have any strong preference between `Div_1` and `DIV1`. I actually may slightly prefer `Div1`, but really I'm OK with any choice.
But I think we should really strive for coherence (and even compatibility when possible) between device series. So I choose DIV1 to match the style used for F4/F7/L1.
From what I see in #58, the choice of `Div_1` was mainly a personal choice. I think coherence is much more important than personal choice.

I'm conscious that this is a breaking change, but it will most likely affect only very few projects and it is very easy to fix. I think it is a small price to pay for long term consistency.

Let me know what you think about this.
If you prefer the `Div1` style (or anything else, really), I can update this PR (and also patch rcc_v2 and rcc_l1 to match this style).

Reply:
I agree with your conclusion.
So here it is, fixed to use the `DivN` style.